src/components/SVGViewer/stories/SVGViewer.stories.tsx

```diff
@@ -32,31 +32,55 @@ const setupMocks = () => {
   };
 };
 
+const fakeClient: API = {
+  // @ts-ignore
+  files: {
+    getDownloadUrls: (): Promise<(FileLink & IdEither)[]> => {
```

Comment:
I guess it's not necessary to define return type here since it's already defined in API interface.